add notes about config caching to config and faq pages (#504)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/commerce-configuration.mdx

</Vocabulary>

## Configuration Caching

The storefront configuration is cached at multiple levels to optimize performance:

> - **CDN Caching**: The configuration file is cached by the EDS CDN with a `max-age=7200` (2 hours) cache control header. This means configuration changes may take up to 2 hours to be reflected on the CDN after being deployed.

Reviewer: ```suggestion
- **CDN Caching**: The configuration file is cached by the EDS CDN with a `max-age=7200` (2 hours) cache control header. This means configuration changes can take up to 2 hours to be reflected on the CDN after deployment.
```

- **Browser Caching**: When the configuration is loaded in the browser, it is stored in [session storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage), which persists until the browser session is cleared.

> **For developers**: After making configuration changes, you may need to wait for the CDN cache to expire (up to 2 hours) and clear your browser's session storage to see updates. You can verify the current configuration by accessing `<site.com>/config.json` directly and checking the `last-modified` header.

Reviewer: ```suggestion
**For developers**: After making configuration changes, it can take up to two hours for the CDN cache to expire. Clear your browser's session storage to see updates. Verify the current cache configuration by accessing `<site.com>/config.json` directly and checking the `last-modified` header.
```
